src/ctc/cctc-tombstone.cc

```diff
@@ -24,11 +24,45 @@
 namespace kaldi {
 namespace ctc {
 
```

Comment:
There should be a comment here briefly explaining what the does; and since this function is not exported, it's good practice to declare it 'static'.